packages/core/src/runtime/compileCSSRules.ts

```diff
@@ -0,0 +1,28 @@
+import { compile, middleware, prefixer, rulesheet, serialize, stringify } from 'stylis';
+
+import { globalPlugin } from './stylis/globalPlugin';
+import { sortClassesInAtRulesPlugin } from './stylis/sortClassesInAtRulesPlugin';
+
+// eslint-disable-next-line @typescript-eslint/no-empty-function
+function noop() {}
+
+export function compileCSSRules(cssRules: string, sortClassesInAtRules = false): string[] {
```

Comment:
why make the sort optional at all? it's not configurable in userland

Reply:
Makes sense, made it required